core/scripts/cli/sponsor/showMarketDetails.js

```diff
@@ -28,15 +29,30 @@ const showMarketDetails = async (web3, artifacts, emp) => {
       "Collateral:",
       fromWei(collateral.toString())
     );
+
+    const actions = {
+      back: "Back",
```

Comment:
do you still need the `backAction` variable declaration on line 11?

Reply:
Do you mind if I delete it in a follow up? Getting this PR in would unblock both of us I guess.

Reply:
sure